CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Changelog
+
+
+## v0.7.0 (TBD)
+
+Starting a Changelog.
+
+### New features and changes
+
+* Dropped Python 2 support. Python >= 3.6 is now required.
+* Added new LicenseListVersion tag.
+* Added `pyspdxtools_convertor` and `pyspdxtools_parser` CLI scripts. See [the readme](README.md) for usage instructions.
```

Comment:
Didn't we just rename them instead of adding them?

Also, we improved the output of the CLI parser (not saying it is perfect now, though)

Reply:
No, they were not included at all in the previous release